spec/pronto/git/repository_spec.rb

```diff
@@ -92,6 +92,11 @@ module Git
           let(:sha) { :staged }
           it { should be_one }
         end
+
+        context 'workdir' do
+          let(:sha) { :workdir }
+          it { subject.count.should eq 48 }
```

Comment:
Can you explain why it should be 48?

Reply:
It's including all the files from the repositories, too:

<details><summary>List of 48 files</summary>

```ruby
["hamlet.txt",
 "message_with_path.xml",
 "message_without_line.xml",
 "message_without_path.xml",
 "new_file.txt",
 "renamed-file.git/config",
 "renamed-file.git/HEAD",
 "renamed-file.git/index",
 "renamed-file.git/logs/HEAD",
 "renamed-file.git/logs/refs/heads/master",
 "renamed-file.git/logs/refs/heads/new_branch",
 "renamed-file.git/objects/20/6ab76efefefb5b457107f9e841ccf06c3db4eb",
 "renamed-file.git/objects/3a/57a3eff4ae575cedf9643a36e61422fe9be74b",
 "renamed-file.git/objects/3e/2cbd49d2d9d33b6c9eea4371eb67c947920f71",
 "renamed-file.git/objects/81/d7ad66f39011d0eedaa70e33bae27b67c3c519",
 "renamed-file.git/objects/93/657bccfddbce46b58c23960fea84103d90d37d",
 "renamed-file.git/objects/97/422ff3a9482718dae96ab9afc6b60c16450076",
 "renamed-file.git/objects/a1/69cecd3261a284b1c3bfd84ad4eb2c7cdcd7e4",
 "renamed-file.git/objects/fd/497adbc2fec28a3a13f49693b7a03dbfe8f542",
 "renamed-file.git/refs/heads/master",
 "renamed-file.git/refs/heads/new_branch",
 "test.git/config",
 "test.git/HEAD",
 "test.git/index",
 "test.git/logs/HEAD",
 "test.git/logs/refs/heads/master",
 "test.git/objects/07/b7623cc56ccfc25ef45a65e4723b3d720504b6",
 "test.git/objects/14/cf9b86fa9661e967c8a526374384a4f10fb5ea",
 "test.git/objects/22/51181228419719f7d926b5326d744ad871ad4a",
 "test.git/objects/3e/0e3ab0a436fc2a9c05253a439dc6084699b7d5",
 "test.git/objects/3f/c0911fde02260ba650ca3bd428f2bb5f88e93f",
 "test.git/objects/56/0c7d57b1ce19e1852c5193e826b62c25f7db4b",
 "test.git/objects/57/7afa184c9bc82a66c40047d0809e5fcc43489f",
 "test.git/objects/5a/6e1531cd6d79810356e7ea7c77485514db28c8",
 "test.git/objects/64/dadfdb7c7437476782e8eb024085862e6287d6",
 "test.git/objects/7b/21c8f4dfb0b8aa39739fc16678c5934877a414",
 "test.git/objects/8f/a4900507f731bb4c827c606facdf8c76aa29ab",
 "test.git/objects/92/8676bf506a52bf693b82a29f45db812e4f18d6",
 "test.git/objects/97/1783b0dbb66e03b18379a8604d2e1bb64cbb01",
 "test.git/objects/97/8cbcccb92bdfa34587dd175f183a2678f0c9b3",
 "test.git/objects/ac/86326d7231ad77dab94e2c4f6f61245a2d9bec",
 "test.git/objects/af/eef802b84cab684cf1b99fdbd444fe57c41141",
 "test.git/objects/d0/e5047bf0930d6cfa151e65bc0aaaa4a7e7de0d",
 "test.git/objects/d6/d56582ebfd0c6c3263ea4c4e2d727048370124",
 "test.git/objects/e6/9de29bb2d1d6434b8b29ae775ad8c2e48c5391",
 "test.git/objects/ec/05bab7d263d5e01be99f2c4e10a5974e24e6de",
 "test.git/objects/ef/4cc92872c2e34ea4b508e024df2b80a2268f1b",
 "test.git/refs/heads/master"]
```

</details>

I'm all ears for a better way to test this, but this was the best I could figure on my own coming into the project for the first time.

Reply:
@ashkulz Are there changes needed here?

Reply:
Sorry, I was busy IRL and didn't see your message. Maybe add a comment on how this came about and I'll merge it ASAP -- sorry for the delay!

Reply:
Added a comment on the source of the count as requested.